srv/processor-service.cds

```diff
@@ -0,0 +1,6 @@
+using { sap.capire.incidents as my } from '../db/schema';
+
+service ProcessorService {
+  entity Incidents as projection on my.Incidents;
+  annotate my.Customers with @cds.autoexpose;
+}
```

Comment:
Hi @danjoa, ProcessorService missing the `support` role. Can you please include it.

Reply:
Ok, I see its moved to `xmpls/authorizations.cds`. Is the idea to copy into srv when required ?

Reply:
Exactly. Allows us to easily demonstrate unrestricted app, and then restricted one by just moving the `xmpls/authorizations.cds` into place, as with the other

Plus I wanted to take the opportunity to promote the separation of concerns as a best practice also for authorisations. Doesn't have to be like that